week_2/pet.rb

```diff
@@ -90,7 +90,7 @@ def food_cost(days)
   # snake = Pet.new(name: 'python', animal_type_id: 4, food_consumed_per_day: 0.3)
   # Pet.cost_to_feed([cat, dog, fish, snake], 6) will return 6180.0
   def self.cost_to_feed(pets, days)
-    raise NotImplementedError # TODO
+    pets.reduce(0){|sum,pet| sum+pet.food_consumed_per_day*days*FOOD_COST_PER_KG[pet.animal_type_id]}
```

Comment:
Keep your code [`DRY`](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself).

```suggestion
    pets.reduce(0) { |sum,pet| sum + pet.food_cost(days) }
```

Reply:
Done